hub/core/chunk.py

```diff
@@ -108,8 +108,10 @@ def update_headers(self, incoming_num_bytes: int, sample_shape: Tuple[int]):
         self.shapes_encoder.add_shape(sample_shape, 1)
         self.byte_positions_encoder.add_byte_position(num_bytes_per_sample, 1)
 
-    def __len__(self):
+    @property
+    def nbytes(self):
```

Comment:
replaced `len` with `nbytes` for all cachables so that cachable classes can have their `len` computed without returning the num bytes.